app/utils.py

```diff
@@ -36,3 +38,12 @@ def construct_indy_proof_request(
         },
     }
     return indy_proof_request
+
+
+def ed25519_verkey_to_did_key(key: str) -> str:
+    """Convert a naked ed25519 verkey to W3C did:key format."""
```

Comment:
Maybe we can add some mechanism to check whether ed25519 verkey is of valid format so if one passes in a junk key by accident or purpose this fails in a desired way

Reply:
Yes that would be a good one. I'm going to open a separate ticket for it